app/src/main/java/com/github/se/signify/ui/screens/home/QuizScreen.kt

```diff
@@ -187,7 +157,7 @@ fun QuizContent(
   Spacer(modifier = Modifier.height(24.dp))
 
   Button(
-      onClick = { if (selectedOption != null) onSubmit(selectedOption!!) },
+      onClick = { if (selectedOption != null) onSubmit(selectedOption) },
```

Comment:
The removal of the non-null assertion operator eliminates the risk of a runtime exception if the assertion fails. So it's safer. Good job!